docs/pages/en-us/Features.md

```diff
@@ -32,5 +32,8 @@ The service is designed for durability as Elasticsearch provides data redundancy
 
 ## High-Availability
 The service is highly available as Elasticsearch replicates and distributes data across it's scalable, durable cluster.  
+
+## Audit Web Service API and Client
```

Comment:
I could be over thinking here is the REST API considered a key feature here? Maybe it is. If so I'd rename the heading hear to something like REST API Interface or something like that to focus on the fact we have provided a REST API interface to send audit events through to Elasticsearch. The only reason I'm questioning this is that the REST API we have for the Job Service was never considered as a key feature so just raising this.